Readme.md

```diff
@@ -1 +1,62 @@
-## Webchat examples with different frameworks
\ No newline at end of file
+# Helpshift web chat examples
+
+This document serves as a guide to understand the purpose of this repository, the frameworks covered, how to navigate the repository and how to contribute
+
+## Purpose
+
+This repository contains the integration of web chat using different frameworks. The goal is to provide you some sample integrations and how to avoid bad integration of web chat in your app.
+
+## Frameworks Covered
+
+This repo includes sample application build with the following frameworks:
++ React
++ Next.js
++ Angular
++ Vue.js
+
+## Repository structure
+
+This repository is organised into separate directories for each framework. Each directory contains the respective sample application and its associated files
+
+| Directory       | Description                            |
+|-----------------|----------------------------------------|
+| `react/`    | Contains files for a React application. |
+| `next/`   | Contains files for a Next.js application. |
+| `angular/`  | Contains files for an Angular application. |
+| `vue/`      | Contains files for a Vue.js application. |
+
+## Navigating the repository
+
+To explore the sample applications, navigate to the respective directory if the frmework you are interested in:
++ React: `/react/`
++ Next.js: `/next/`
++ Angular: `/angular/`
++ Vue.js: `/vue/`
+
+Each directory contains it's own `README.md` file with specific instructions on how to run the sample app locally.
+
+## Explore the codebase
+
+In our codebase, we use `@NOTE` to highlight the important comments and explanations. These `@NOTE` comments provide additional context and information related to the code. You can find them throughout the codebase of each sample app to understand the logic and functionality.
+
+## How to contribute
+
+If you are making changes to repository, it is possible to use the Github code editor directly for small changes. For larger changes, consider going local:
+
++ Fork the repository
++ Clone the forked repository
++ Create a new branch:  It’s important to create a new branch for each feature or fix you are working on.
+  ```
+   git checkout -b feature/your-feature-name
+   ```
++ Make your changes: Add your changes or new features
++ Commit your changes
+    ```
+    git add .
+    git commit -m "Add your commit message"
+    ```
++ Push your changes
+     ```
+    git push origin feature/your-feature-name
+    ```
++ Create a PR (Pull Request)
```

Comment:
This is the standard process to contribute on any repository, and we shouldn't explain it here.
We can simply say:

Fork this project, make your changes, and open a PR to contribute to this repository.
You can contribute by:
- Updating the existing framework integrations
- Creating sample integrations for other frameworks such as Svelte, Remix, Nuxt.js, etc.